app/assets/stylesheets/src/layout/table.css.scss

```diff
@@ -38,18 +38,44 @@
   }
 }
 
-.modal-body{
+.resources{
   font-family: $font;
   font-size: 2rem;
   text-align: left;
-  width: 100%;
+  background-color: $background_table;
+  border: 0.1rem solid $secondary_color;
+  padding: 1rem;
+  width: 50rem;
+  .Error_title{
+    color: $font_color;
+    margin-top: 1rem;
+  }
+  .title{
+    color: $font_color;
+  }
+  ul{
+    list-style: none;
+  }
+  ul li::before {
+    content: "\2022";
+    color: $secondary_color;
+    margin-left: -1rem;
+  }
+  li{
+    color: $font_color;
+    span{
+      padding-left: 1rem;
+    }
+  }
   .form{
+    width: 100%;
     display:grid;
-    gap:0rem;
+    gap:1rem;
     grid-template-columns: 1fr;
     .form__label{
-      color: black;
+      color: $font_color;
       padding: 1rem;
+      width: 100%;
     }
     .form__input{
       background-color: $background_table;
```

Comment:
Modify this blocks with Bem nomeclature, please, here have a [references](https://andrew-barnes.medium.com/bem-and-sass-a-perfect-match-5e48d9bc3894)